std/traits.d

```diff
@@ -7668,6 +7668,11 @@ template getUDAs(alias symbol, alias attribute)
 {
     import std.meta : Filter;
 
+    alias getUDAs = Filter!(isDesiredUDA!attribute, __traits(getAttributes, symbol));
+}
+
```

Comment:
Hmm Ddoc doesn't seem to like this: http://dtest.dlang.io/diff/website-5d84a318ec802dc2fdc00d2f5a8b7565167d489c-f505cb0447f0cc98330d0ee0ce1c53b4/website-5d84a318ec802dc2fdc00d2f5a8b7565167d489c-06d63a0531f58a1b7a505cb68dc55acc/web/phobos-prerelease/std_traits.html

Reply:
As fixing Ddoc is probably out of reach for the PR, other ideas:
- try whether `/// ditto` works
- move the `private` impl below the public test